src/components/Layouts/_index.scss

```diff
@@ -0,0 +1,104 @@
+////
+/// Modular layouts styles.
+/// @group modular-layouts
+/// @copyright IBM Security 2020
+////
+
+@import '@carbon/colors/scss/colors';
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/theme-maps';
+
+/// Generate a map of spacing tokens from the provided Carbon theme map.
+/// @access private
+/// @param {Map} $theme-map - Map of theme variables.
+/// @returns {Map} - The filtered map of only spacing variables.
+@function get-spacing-tokens-map($theme-map) {
+  $-map: ();
+
+  @each $token, $value in $theme-map {
+    @if type-of($value) == 'number' {
+      $-map: map-merge(
+        $-map,
+        (
+          $token: $value,
+        )
+      );
+    }
+  }
+
+  @return $-map;
+}
+
+/// Map of spacing tokens.
+/// @type Map<String, Number>
+/// @access private
+$spacing-tokens-map: get-spacing-tokens-map($carbon--theme--g100);
```

Comment:
Unless we foresee `get-spacing-tokens-map` being used elsewhere, I recommend calling it inline to promote the practice of only creating functions in situations where code is duplicated, with the exception of if the piece is being tested.

I remember you mentioning you'd like to introduce some tests for `get-spacing-tokens-map` to ensure that the correct values are being returned - is this still the case?